sspl_test/framework/utils/conf_utils.py

```diff
@@ -160,7 +160,7 @@
 CNTRLR_SECONDARY_IP_KEY = "storage_enclosure>%s>controller>secondary>ip" % ENCLOSURE
 CNTRLR_SECONDARY_PORT_KEY = "storage_enclosure>%s>controller>secondary>port" % ENCLOSURE
 CNTRLR_USER_KEY = "storage_enclosure>%s>controller>user" % ENCLOSURE
-CNTRLR_PASSWD_KEY = "storage_enclosure>%s>controller>password" % ENCLOSURE
+CNTRLR_SECRET_KEY = "storage_enclosure>%s>controller>password" % ENCLOSURE
```

Comment:
storage_enclosure>{enclosure-id}>controller>secret

why here secret word not incorporated in key?

Reply:
Missed to update here. But its done in sspl_test and so test were passed.
With new commit. This is also addressed.